### PR TITLE
make a file structure

### PR DESCRIPTION
**Pull request by ismail2009**
#6

### CODE REVIEW

**Review by marwa7med — APPROVED** [review]

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const path = require("path");
+
+module.exports = {
+    "entry": "./src/frontEnd/index.js",
```
Comment (marwa7med) [inline webpack.config.js]: edit this line "./src/client/index.js"

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                ]
+            },
+            {
+                "test": /\.(png|jpg|gif)$/,
+                "use": [
+                    {
+                        "loader": "file-loader",
+                        "options": {
+                            "name": "[path][name].[ext]",
+                            "context": path.resolve("src/frontEnd/assets/")
```
Comment (marwa7med) [inline webpack.config.js]: and this line

**Review by marwa7med — APPROVED** [review]

**Review by NadiaKhatib — APPROVED** [review]

**Review by abdhalees — CHANGES_REQUESTED** [review]
remove `styles` folder from `public` since you're using react and making the styles in `client` folder

**Comment by ismail2009** [comment]
I move it @abdhalees

**Review by abdhalees — APPROVED** [review]

**Pull request merged**